packages/editor-ui/src/components/mixins/nodeHelpers.ts

```diff
@@ -47,6 +48,25 @@ export const nodeHelpers = mixins(
 				return node.type === HTTP_REQUEST_NODE_TYPE && node.typeVersion === 2;
 			},
 
+			isCustomApiCallSelected (nodeValues: INodeParameters): boolean {
```

Comment:
if this is only used in one place, no point in it being in a mixin here. move it into ocmponent

Reply:
Used in `NodeSettings` and `ParameterInputList`